package.json

```diff
@@ -22,28 +22,57 @@
     "start": "skyux serve"
   },
   "peerDependencies": {
-    "@angular/common": "^4.0.0",
-    "@angular/compiler": "^4.0.0",
-    "@angular/core": "^4.0.0",
-    "@angular/forms": "^4.0.0",
-    "@angular/platform-browser": "^4.0.0",
-    "@angular/platform-browser-dynamic": "^4.0.0",
-    "@angular/router": "^4.0.0",
-    "@blackbaud/skyux": "^2.0.0"
+    "@angular/common": ">=4.0.0",
```

Comment:
Is Stache importing from all of these packages directly (for all files within the `./src/app/public` directory)? If so, I would list the version numbers at the lowest possible supported version, so that the majority of SKY UX 3 SPAs can consume Stache without incurring a "missing peer" warning during `npm install`.

For example, does Stache work with `@skyux/core@3.0.0`? If so, you would list it as `^3.0.0`.